person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] Great job so far 👏🏻👏🏻 but I think when you want to create aretal you need a person and a book as well with a specific date so firstly in this personfile you will need to pass book and date here then creating a new rental so kindly associate the person with a book when creating a rental 💯🔥✅🚀🙏🏻✨